prov/util/src/rxm_av.c

```diff
@@ -119,14 +124,20 @@ util_get_peer(struct rxm_av *av, const void *addr)
 	return peer;
 }
 
+void _util_put_peer(struct util_peer_addr *peer)
+{
+	/* assert(ofi_mutex_held(&av->util_av.lock)); */
```

Comment:
See above comment.  But also see below, as the 2 new functions likely aren't needed.